receiver/awsfirehosereceiver/internal/unmarshaler/cwlog/cwlog.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package cwlog
+
+type cwLog struct {
```

Comment:
If we want to match the existing `cWMetric` type this should be `cWLog`. I'm not advocating one format over the other, just suggesting uniformity either way.